_episodes/02-make-it-public.md

```diff
@@ -315,3 +315,31 @@ The first command will download the correct image from the repository (the `fpso
 To complete these steps, you can see your running containers using the `docker ps` command. To see **all** containers, add the `-a` flag - `docker ps -a`.
 
 ## What if your software is an analysis in a Notebook?
+
+### What is Jupyter?
+
+Jupyter notebook, formerly known as IPython (or Interactive Python), is a flexible and powerful open source research tool that can help you keep a narrative of your coding process. The name Jupyter is an acronym of the three core languages it was designed for: **JU**lia, **PYT**hon, and **R**. [Project Jupyter](http://jupyter.org/) supports interactive data science and scientific computing across more than 40 programming languages. There are a lot of resources that can help you better understand how to prepare and structure a Jupyter notebook, such as the [official Documentation](http://jupyter.org/documentation) and the [Introduction materials for Reproducible Research Curriculum](https://github.com/Reproducible-Science-Curriculum/introduction-RR-Jupyter).
+
+Jupyter is one type of computational notebooks. Another type that shares the same philosophy and overall structure (but has a different syntax) is the RMarkdown documents that are particularly prevalent for R-specific projects.
+
+### How can I make my notebook public?
+
+Sharing the notebook file itself (as a `ipynb` or `rmd` file) is the exact same as sharing any other piece of code; create a repository with the correct structure in a version control system (such as GitHub). However, there are a few additional steps that one might take to improve the reproducibility and accessibility of the notebook.
```

Comment:
(such as Git) GitHub is not a version control system

Reply:
Done!